frame/ethereum/src/mock.rs

```diff
@@ -115,9 +117,20 @@ impl pallet_evm::Trait for Test {
 	type Precompiles = ();
 }
 
+pub struct EthereumFindAuthor;
+impl FindAuthor<H160> for EthereumFindAuthor {
+	fn find_author<'a, I>(_digests: I) -> Option<H160> where
+		I: 'a + IntoIterator<Item=(ConsensusEngineId, &'a [u8])>
+	{
+		None
```

Comment:
I think this is still TODO, right?

Reply:
Just updated so it returns a "real" address. Still no tests associated to this particular implementation though.